services/notifications/Dockerfile

```diff
@@ -57,6 +56,8 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
 # those from our virtualenv.
 ENV PATH="${VIRTUAL_ENV}/bin:$PATH"
 
+EXPOSE 8000
```

Comment:
can you keep it the same as in all the other Dockerfiles? thanks
I spent too much time already to have the Dockerfiles the same everywhere for easier maintenance. thanks.